tests/ui/let_with_type_underscore.fixed

```diff
@@ -26,7 +26,7 @@ fn main() {
     let x = func();
     let x: Vec<_> = Vec::<u32>::new();
     let x: [_; 1] = [1];
-    let x = 1;
+    let x  = 1;
```

Comment:
This is an unfortunate side effect. Could you make it also swallow whitespaces before the `:` (look for example for `with_leading_whitespace()`)?